templates/_homepage.html

```diff
@@ -1,9 +1,4 @@
-{% load emojificate %}
-{% load humanize %}
-{% load i18n %}
-{% load package_tags %}
-{% load typogrify_tags %}
-{% load waffle_tags %}
+{% load emojificate humanize i18n package_tags typogrify_tags waffle_tags %}
```

Comment:
I personally like each load tag on a separate line.

Reply:
@saadmk11 I agree. There is a conversation in the [forum](https://forum.djangoproject.com/t/django-template-style-guide-extensions/35351) about this with some people wanting them on a single line alphabetized, with others advocating for a single line per library. 

There's also an [issue in djade](https://github.com/adamchainz/djade/issues/33) with a discussion about this very thing and @jefftriplett [indicating a preference for one load-statement per line](https://github.com/adamchainz/djade/issues/33#issuecomment-2386226216) because

> it streamlines code reviews and diffs.

It could be that options are added to `djade` to allow for this preference but that's still unclear

Reply:
Same, I think leaving a message on the post @ryancheley highlighted is a good place to mention that too. 

While I like that Black has only one configuration option, this is the one I hope `djade` adopts so that people have a choice.